tests/define-arrow.test.js

```diff
@@ -192,6 +192,19 @@ describe('Plugin for define blocks with arrow function factories', () => {
     `);
   });
 
+  it('does not require a dependency named `require` (ugly)', () => {
```

Comment:
This case is not necessarily specific to uglification, so let's replace these test titles with "... which has been renamed" instead of "... (ugly)".